rsbuild/ssr-express-with-manifest/server.mjs

```diff
@@ -0,0 +1,69 @@
+import express from "express";
+import fs from 'node:fs';
+import { createRsbuild, loadConfig, logger } from "@rsbuild/core";
+
+const templateHtml = fs.readFileSync('./template.html', 'utf-8');
+
+const serverRender = (serverAPI) => async (_req, res) => {
+  const indexModule = await serverAPI.environments.ssr.loadBundle("index");
+
+  const markup = indexModule.render();
+
+  const { entries } = JSON.parse(fs.readFileSync('./dist/manifest.json', 'utf-8'));
```

Comment:
Maybe we should support `await serverAPI.environments.web.readFile("manifest.json")` API?

Reply:
Agreed, we can design a new API for this scenario.